apps/editor.planx.uk/src/pages/FlowEditor/components/NewSettings/TeamSettings/IntegrationSettings/SubmissionEmails/types.ts

```diff
@@ -0,0 +1,16 @@
+export interface SubmissionEmailFormValues {
+  submissionEmail: string;
+  defaultEmail: boolean;
+}
+
+export interface GetTeamSubmissionIntegrationsData {
+  submissionIntegrations: {
+    submissionEmail: string;
+    defaultEmail: boolean;
+  }[];
+}
+
+export interface UpdateTeamSubmissionIntegrationsVariables {
+  submissionEmail: string;
+  defaultEmail: boolean;
+}
```

Comment:
```suggestion
export interface UpdateTeamSubmissionIntegrationsVariables {
  teamId: number;
  submissionEmail: string;
  defaultEmail: boolean;
}
```

We also need to pass this value in to the mutation. It's a bit tricky atm because there's no hard link between GQL queries and their associated types but we'll resolve this (in the medium-ish?) term via codegen.

Reply:
Please see comment below!